### ISSUE TITLE
AutoType doesn't work on MacOS + Chrome when Signal desktop client is running

### ISSUE THREAD

**Issue by sts10** [keepassxreboot/keepassxc#611]
On a Mac, I'm unable to AutoType into the Chrome browser while the Signal messaging desktop client is open and running.

## Expected Behavior
I'd like to be able to AutoType a username and password into a Chrome tab while I'm also running [Signal's desktop messenger](https://chrome.google.com/webstore/detail/signal-private-messenger/bikioccmkafdpakkkcpdbppfkghcmihk?hl=en). As I understand it, this desktop client runs as a Chrome app.

## Current Behavior
Currently, if I have Signal's desktop client open, AutoType types my username and password into my Signal messenger message text box or in the Signal's client username search box, rather than the open Chrome tab that's waiting for a username and password. This happens even if the Chrome tab was the last thing I had in focus. 

## Possible Solution
I'd guess that this is because the Signal desktop client is a Chrome app, which apparently confuses KeePassXC's AutoType feature. FYI, I _can_ AutoType into a login form on Firefox while the Signal desktop client is open.

## Steps to Reproduce (for bugs)
1. Open Chrome browser.
2. If you haven't already, download and set up/sign in to [Signal's desktop client](https://chrome.google.com/webstore/detail/signal-private-messenger/bikioccmkafdpakkkcpdbppfkghcmihk?hl=en)
3. In the Chrome browser, go to a log in page like [Reddit's](https://www.reddit.com/login)
4. Switch over to KeePassXC and attempt to execute an AutoType for your Reddit credentials
5. Your email address that you login to Reddit with will be AutoTyped into the user search box on Signal, rather than the username text box on Reddit.

## Context
This isn't a huge thing for me, as I don't run Signal's desktop client all the time. But I could imagine that some privacy-minded folks who would be inclined to use KeePassXC also use Signal frequently, and thus have it open constantly. It'd be a bummer if they couldn't use AutoType in Chrome. There's also the risk that they accidentally send their username and/or password to a Signal contact when attempting an AutoType. 

## Your Environment
* KeePassXC version/commit used: 2.1.4 `cb283bb95ae559d044415f426f841c27bff383f0` (installed via the DMG on the KeePassXC website)
* Qt version (e.g. Qt 5.3): Qt 5.8.0
* Chrome version: 58.0.3029.110 (64-bit)
* Operating System and version: Mac OS 10.10.5

**Comment by droidmonkey** [comment]
This is probably because the signal app is trapping keyboard combinations inside Chrome. The same thing happened to me when I had them enabled for the Google Music extension.

http://www.omgchrome.com/chrome-google-music-media-keys/

**Comment by sts10** [comment]
Ah bummer. 

One work-around I've found already is that if you minimize Signal first, AutoType appears to work as you'd expect/want. Maybe w'ell just have to deal with this one.

**Comment by weslly** [comment]
@sts10 do you have the "Always ask before performing auto-type" option enabled?

**Comment by sts10** [comment]
Hm this is strange. 

First: I did indeed have the "Always ask before performing auto-type" option enabled, though it wasn't asking me (see #579 ). 

But now, strangely, I can't reproduce this issue at all, with the option checked or unchecked. Now the Auto-Type in Chrome works correctly and as expected, with the Signal app running. 

This is obviously good news-- and we might be able to close the issue-- but I'm not sure why it's working now. I guess there's a chance either Chrome or Signal got updated and that did it? (I'm still using KeePassXC 2.1.4 from the DMG, and I don't think I changed any settings.) If that's the case (if no one else can reproduce this any longer), I'll leave it to y'all whether to close this issue or not, and sorry for wasting some time.
